hr_timesheet_portal/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Timesheet portal (editable)",
+    "summary": "Fill in timesheets via the portal",
+    "version": "12.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Website",
+    "website": "https://github.com/OCA/hr-timesheet",
+    "author": "Hunki Enterprises BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "hr_timesheet",
```

Comment:
I think this should include `portal` and `website` still, that would also solve Travis errors